tests/Unit/Entity/EditableOrderTraitTest.php

```diff
@@ -0,0 +1,36 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Setono\SyliusOrderEditPlugin\Tests\Unit\Entity;
+
+use PHPUnit\Framework\TestCase;
+use Setono\SyliusOrderEditPlugin\Tests\Application\Entity\Order;
+
+final class EditableOrderTraitTest extends TestCase
+{
+    /**
+     * @test
+     */
```

Comment:
```suggestion
    /** @test */
```

I personally like to make them one-liners, to occupy less space :dancer: